plugins/jobs/web_client/templates/jobCheckBoxMenu.pug

```diff
@@ -2,10 +2,16 @@ a.g-job-checkbox-menu-link(data-toggle="dropdown", data-target=`#g-job-checkbox-
   | #{title}
   i.icon-filter
 .dropdown(id=`g-job-checkbox-menu-${title}`)
-  ul.dropdown-menu(role="menu")
-    each checked, name in values
-      li(role="presentation")
-        label
-          input.g-job-filter-checkbox(type="checkbox", data-toggle="tooltip",
-              id=name, title=name, checked=checked)
-          span #{name}
+  .dropdown-menu(role="menu")
+    .checkbox
+      label
+        input.g-job-filter-checkall(type="checkbox", data-toggle="tooltip", checked=checkAllChecked)
+        | Check all
+    hr
+    ul
+      each checked, name in values
+        li(role="presentation")
+          .checkbox
+            label
+              input.g-job-filter-checkbox(type="checkbox", data-toggle="tooltip", id=name, title=name, checked=checked)
+              | #{name}
```

Comment:
There's a shorthand syntax for this in pug, this is equivalent to:

    = name